server/views/pages/dashboardRecat.html

```diff
@@ -24,8 +24,31 @@ <h1 class="govuk-heading-l">Dashboard - Categorisation reviews</h1>
 <div>
   {% include "../partials/dashboardForm.html" %}
   <hr/>
+  <h3>Categorisation review numbers</h3>
```

Comment:
a cycler is a nunjucks function which cycles round, returning each of its values in turn